src/routes/safe/components/Balances/Coins/index.tsx

```diff
@@ -84,16 +73,16 @@ const Coins = (props: Props): React.ReactElement => {
   const activeTokens = useSelector(extendedSafeTokensSelector)
   const currencyValues = useSelector(safeFiatBalancesListSelector)
   const granted = useSelector(grantedSelector)
-  const [filteredData, setFilteredData] = React.useState<List<BalanceData>>(List())
   const { trackEvent } = useAnalytics()
 
   useEffect(() => {
     trackEvent({ category: SAFE_NAVIGATION_EVENT, action: 'Coins' })
   }, [trackEvent])
 
-  useMemo(() => {
-    setFilteredData(getBalanceData(activeTokens, selectedCurrency, currencyValues, currencyRate))
-  }, [activeTokens, selectedCurrency, currencyValues, currencyRate])
+  const filteredData: List<BalanceData> = React.useMemo(
```

Comment:
Nitpick: we use effect via import, but useMemo is used via accessing it on React namespace. Could we make it consistent?